service/controller/azure/v6/resource_set.go

```diff
@@ -130,6 +130,8 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		c := guestcluster.Config{
 			CertsSearcher: config.CertSearcher,
 			Logger:        config.Logger,
+
+			CertID: certs.ClusterOperatorAPICert,
```

Comment:
Another change that was made in the library. Ensures we search for the correct cert.